downstream/modules/platform/ref-controller-node-counting.adoc

```diff
@@ -2,11 +2,11 @@
 
 = Node counting in licenses
 
-The {ControllerName} license defines the number of Managed Nodes that can be managed as part of a {PlatformName} subscription.
+The {PlatformNameShort} license defines the number of Managed Nodes that can be managed as part of your subscription.
 
 A typical license says "License Count: 500", which sets the maximum number of Managed Nodes at 500.
 
-For more information about managed node requirements for licensing, see https://access.redhat.com/articles/3331481.
+For more information about managed node requirements for licensing, see link:https://access.redhat.com/articles/3331481[How are "managed nodes" defined as part of the {PlatformName} offering]
```

Comment:
Period at the end of the sentence.

Reply:
Fixed in https://github.com/ansible/aap-docs/pull/1883